bazel/repositories.bzl

```diff
@@ -44,6 +44,19 @@ def proxy_wasm_cpp_host_repositories():
         url = "https://github.com/bytecodealliance/wasmtime/archive/v0.26.0.tar.gz",
     )
 
+    http_archive(
+        name = "wamr",
+        build_file = "@proxy_wasm_cpp_host//bazel/external:wamr.BUILD",
+        sha256 = "f819b9ec866a12086233e578044dd8297e6be86f0f17807b16124f78a3652d4f",
+        url = "https://github.com/lum1n0us/wasm-micro-runtime/releases/download/WAMR-01-29-2021/source.zip",
+    )
```

Comment:
Could you move this before `wasmtime`? You've added this in between `wasmtime` and `wasm_c_api` (used by Wasmtime), so it's easier to miss during updates.

Reply:
OK